game-core/src/main/java/games/strategy/triplea/ui/menubar/help/UnitStatsTable.java

```diff
@@ -0,0 +1,93 @@
+package games.strategy.triplea.ui.menubar.help;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.data.ResourceCollection;
+import games.strategy.engine.data.UnitType;
+import games.strategy.triplea.image.UnitImageFactory;
+import games.strategy.triplea.ui.TooltipProperties;
+import games.strategy.triplea.ui.UiContext;
+import games.strategy.triplea.util.TuvUtils;
+import java.net.URL;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class UnitStatsTable {
+
+  public static String getUnitStatsTable(final GameData gameData, final UiContext uiContext) {
+    // html formatted string
+    final StringBuilder hints = new StringBuilder();
+    hints.append("<html>");
+    hints.append("<head><style>th, tr{color:black}</style></head>");
+    try {
+      gameData.acquireReadLock();
+      final Map<GamePlayer, Map<UnitType, ResourceCollection>> costs =
+          TuvUtils.getResourceCostsForTuv(gameData, true);
+      final Map<GamePlayer, List<UnitType>> playerUnitTypes =
+          UnitType.getAllPlayerUnitsWithImages(gameData, uiContext, true);
+      final String color3 = "FEECE2";
+      final String color2 = "BDBDBD";
+      final String color1 = "ABABAB";
+      int i = 0;
+      for (final Map.Entry<GamePlayer, List<UnitType>> entry : playerUnitTypes.entrySet()) {
+        final GamePlayer player = entry.getKey();
+        hints.append("<p><table border=\"1\" bgcolor=\"" + color1 + "\">");
+        hints
+            .append(
+                "<tr><th style=\"font-size:120%;000000\" bgcolor=\"" + color3 + "\" colspan=\"4\">")
```

Comment:
Unrelated too, I just don't get it though
```suggestion
                "<tr><th style=\"font-size:120%;\" bgcolor=\"" + color3 + "\" colspan=\"4\">")
```